### DISCUSSION TITLE
가상 메모리 | Jae’s Blog

### DISCUSSION THREAD

**Discussion by giscus** [jaenam615/jaenam615.github.io#13] [Announcements]
# 가상 메모리 | Jae’s Blog

Hydejack is a boutique Jekyll theme for hackers, nerds, and academics, with a focus on personal sites that are meant to impress.

https://jaenam615.github.io/cs/os/2024-03-22-virtual_memory/

**Comment by moonmisae118** [comment]
가상 메모리 내용은 언제 올라오나요!

**Reply by yunsejin** [reply]
금방 올라갑니다~
